packages/core/src/compiler.ts

```diff
@@ -74,7 +74,8 @@ export default class Compiler {
     if (typeof src === 'object') {
       const result: StringStructure = {};
       for (const key of Object.keys(src)) {
-        const pl = (plurals && plurals[key]) || plural;
+        const normalizedPluralsKey = key.length > 2 ? normalize(key) : key;
+        const pl = (plurals && plurals[normalizedPluralsKey])  || plural;
```

Comment:
I considered initially making this a double fallback:

```
const pl = (plurals && plurals[normalizedPluralsKey]) || (plurals && plurals[key]) || plural;
```

But could not determine whether this covers a desirable fallback path. Could use guidance here.

Reply:
if 1) normalization is to be trusted, and 2) only normalized values are to be supported, then this appears to be a safe refactor